src/main/java/io/strimzi/kafka/bridge/http/HttpSinkBridgeEndpoint.java

```diff
@@ -442,4 +442,26 @@ private boolean checkAcceptedBody(String accept) {
         }
         return false;
     }
+
+    /**
+     * Build the request URI for the future consumer requests
+     *
+     * @param routingContext context of the current HTTP request
+     * @return the request URI for the future consumer requests
+     */
+    @SuppressWarnings({"checkstyle:BooleanExpressionComplexity"})
+    private String buildRequestUri(RoutingContext routingContext) {
+        String baseUri = routingContext.request().absoluteURI();
+
+        String xForwardedHost = routingContext.request().getHeader("x-forwarded-host");
```

Comment:
No support for `Forwarded`?

Reply:
Well, do we really need that? We have to parse for "host" and "proto" inside it and anyway the proxy has to add `X-Forwarded-Port` which is something not defined. What's the advantage of supporting `Forwarded` + `X-Forwarded-Port` instead of having one more header?

Reply:
If you read the documentation carefully you'll see that the port would be included in the `host`, since the `Forwarded.host` is defined to be the value of the `Host` header observed by the proxy, and `Host` is defined to include both hostname and port.

Reply:
You are right but the TCP port is marked as optional. If not given it implies the default for the service requested, so we should default to 80 for HTTP and 443 for HTTPS.
Anyway, we need some more process on the "host" we get.

Reply:
It just seems strange to me that we're adding support for non-standard `X-` headers without adding support for the standardized header which is intended to replace the use of the non-standard versions. RFC7239 isn't even particularly new, it's half a decade old.